go.mod

```diff
@@ -6,32 +6,130 @@ go 1.17
 
 require (
 	github.com/Axway/agent-sdk v1.1.18-0.20220322223119-4c2f1f99b378
+	github.com/elastic/beats/v7 v7.7.1
+	github.com/getkin/kin-openapi v0.67.0
+	github.com/sirupsen/logrus v1.8.1
+	github.com/stretchr/testify v1.7.0
+	github.com/tidwall/gjson v1.13.0
+	gopkg.in/yaml.v2 v2.4.0
+	sigs.k8s.io/yaml v1.2.0
+)
+
+require (
```

Comment:
not sure why it ended up with 2 require sections

Reply:
Looks like go 1.17 make a require section for indirect dependencies